### PR TITLE
Doc Queue #58: Add note to vmware instructions about configuring jail networking

### PR DESCRIPTION
**Pull request by Mrt134**
HTML build test: no issues.

### CODE REVIEW

**Comment by Mrt134** [comment]
Waiting to merge until QE can confirm instruction.

File: userguide/install.rst [hunk 1]
```diff
@@ -1114,3 +1114,10 @@ click :menuselection:`Options --> Advanced --> File Locations`.
 Locate the path for the Configuration file named :file:`filename.vmx`.
 Open that file in a text editor, change :guilabel:`hpet0.present` from
 *true* to *false*, and save the change.
+
+If plugins or jails inside the %brand% VM are encountering network
```
Comment (a-stjohn) [inline userguide/install.rst]: This is a lengthy "if-then" sentence. It may be fine in this case though.

Comment (wblock) [reply userguide/install.rst]: It's not terrible with just the link text, the URLs make it look much longer.

File: userguide/install.rst [hunk 2]
```diff
@@ -1114,3 +1114,10 @@ click :menuselection:`Options --> Advanced --> File Locations`.
...
+connectivity issues, edit the connected
```
Comment (wblock) [inline userguide/install.rst]: Any connectivity issues, or just specific ones?

**Pull request merged**